cli/common/utils.go

```diff
@@ -237,3 +237,45 @@ Loop:
 
 	return lastNewLinePos, nil
 }
+
+func ConvertToStringsSlice(s interface{}) ([]string, error) {
```

Comment:
Could it be declared as `s []interface{}`?

Reply:
No, since it should be converted `interface{} -> []interface{} -> []string`